woocommerce/woocommerce-android
# [Themes] - Theme preview

Completes #10259. The PR adds the preview options for Mobile, Tablet and Desktop and changes the WebView scaling based on the choice. 

It is critical that the WebView is reloaded after the scaling is changed, otherwise the viewport size gets changed and the whole pages gets zoomed in/out without a way to change it.

[Screen_recording_20231130_164915.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/755a880c-f9d4-4340-9689-c96214fee189)

**To test:**
1. Go to Settings -> Themes
2. Tap on some theme
3. Notice a preview is opened (mobile view by default)
4. Try tapping on the preview type menu and change it to Tablet or Desktop
5. Notice the WebView scaling is updated to reflect the preview type
